docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplate.java

```diff
@@ -382,6 +389,15 @@ public FormValidation doCheckVolumesFromString(@QueryParameter String volumesFro
             return FormValidation.ok();
         }
 
+        @Restricted(DoNotUse.class)
+        public List<Descriptor<RetentionStrategy<?>>> getDockerRetentionStrategies() {
+            List<Descriptor<RetentionStrategy<?>>> strategies = new ArrayList<Descriptor<RetentionStrategy<?>>>();
+            strategies.addAll(RetentionStrategy.all());
+            strategies.add(DockerOnceRetentionStrategy.DESCRIPTOR);
+            strategies.add(DockerCloudRetentionStrategy.DESCRIPTOR);
```

Comment:
No, is there any desired order? It should be trivial to create any order you like this way.